### ISSUE TITLE
Error with basic-tf-example

### ISSUE THREAD

**Issue by sestegra** [getporter/terraform-mixin#56]
### Environment
```
macos 10.15.6
docker 19.03.13
porter v0.29.1 (2e0f3648)
terraform mixin v0.6.0
```

### Scenario
```
$ cd examples/basic-tf-example
$ porter build
Copying porter runtime ===>
Copying mixins ===>
Copying mixin terraform ===>

Generating Dockerfile =======>

Writing Dockerfile =======>

Starting Invocation Image Build =======>
$ porter install
porter install
installing basic-tf-example...
executing install action from basic-tf-example (installation: basic-tf-example)
Install Terraform assets
Initializing Terraform...
/usr/bin/terraform terraform init

Initializing the backend...

Initializing provider plugins...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.local: version = "~> 1.4"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
stat rocks!: no such file or directory
Error: error running command terraform apply -auto-approve -input=false -var myvar=porter rocks!: exit status 1
err: error running command terraform apply -auto-approve -input=false -var myvar=porter rocks!: exit status 1
Error: mixin execution failed: exit status 1
Error: 2 errors occurred:
	* container exit code: 1, message: <nil>. fetching outputs failed: error copying outputs from container: Error: No such container:path: eefa835f1f0b5e32b94330c7e5bb0817093710c6c172c7a89fe1348bfe1bfdca:/cnab/app/outputs
	* required output myvar is missing and has no default
```

### Results
Terraform variables with spaces are not surrounded by quotes.
```
terraform apply -auto-approve -input=false -var myvar=porter rocks!
```

### Investigation
In following source code, quotes should be added:
https://github.com/getporter/terraform-mixin/blob/5f85f7071b6d81662476746cc4d116e72710d7ac/pkg/terraform/install.go#L32

**Comment by sestegra** [comment]
Error fixed with `v0.6.0-21-g174d233` canary version in commit 12b0f573dd52ff0caf0db1131b64eb864c020883.

**Comment by carolynvs** [comment]
@sestegra Thank you for giving us a heads up! 🙇‍♀️  The problem was that we fixed the mixin but forgot to release an updated version. 🤦‍♀️ [v0.6.1](https://github.com/getporter/terraform-mixin/releases/tag/v0.6.1) now has the fix in it
